### DISCUSSION TITLE
Next Pre Release Date

### DISCUSSION THREAD

**Discussion by genestrashcan** [BEEmod/BEE2-items#4195] [Tech Support / Q&A] [answered]
When will the next pre release come out, it has been almost 6 months since the last one...

**Comment by LautaroL20** [comment, marked as answer]
To make a new release to worth it i would need to solve this issue: https://github.com/BEEmod/BEE2-items/issues/2773 which it will take some time because all styled instances needs to be fixed separatedly thru hammer.

**Comment by LautaroL20** [comment]
However, the most memorable part of the next release will be the implementation of the following feature: https://github.com/BEEmod/BEE2.4/issues/216

**Comment by vrad-exe** [comment]
See [here](https://github.com/BEEmod/BEE2-items/wiki/Upcoming-features#implemented-for-next-release) for a full list of upcoming features. To answer the original question, there isn't a definite release date. The next version will be released when it's ready.
